ui/src/app/pages/analysis/util.ts

```diff
@@ -27,15 +27,21 @@ export function appendJupyterNotebookFileSuffix(filename: string) {
 }
 
 export function appendRStudioNotebookFileSuffix(filename: string) {
```

Comment:
These methods all look like they have the desired behavior now:
```
appendRScriptSuffix('foo.r') -> 'foo.r'
appendRScriptSuffix('foo') -> 'foo.R'
appendAnalysisFileSuffixByOldName('blah', 'oldname.r') -> 'blah.R' (canonical capitalization)
appendAnalysisFileSuffixByOldName('blah.r', 'oldname.r') -> 'blah.r' (given capitalization)
```
but I am not sure that the (existing) function names are quite right - at least I'm finding something to be unintuitive here.  Would you mind trying to come up with comments or new names for these that would help?